iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerWithEngineBase.java

```diff
@@ -108,57 +108,46 @@ public abstract class HiveIcebergStorageHandlerWithEngineBase {
       StatsSetupConst.TOTAL_SIZE, SnapshotSummary.TOTAL_FILE_SIZE_PROP
   );
 
-  @Parameters(name = "fileFormat={0}, engine={1}, catalog={2}, isVectorized={3}, formatVersion={4}")
+  @Parameters(name = "fileFormat={0}, catalog={1}, isVectorized={2}, formatVersion={3}")
   public static Collection<Object[]> parameters() {
     Collection<Object[]> testParams = Lists.newArrayList();
-    String javaVersion = System.getProperty("java.specification.version");
 
     // Run tests with every FileFormat for a single Catalog (HiveCatalog)
     for (FileFormat fileFormat : HiveIcebergStorageHandlerTestUtils.FILE_FORMATS) {
-      for (String engine : EXECUTION_ENGINES) {
-        IntStream.of(2, 1).forEach(formatVersion -> {
-          // include Tez tests only for Java 8
-          if (javaVersion.equals("1.8")) {
-            testParams.add(new Object[]{fileFormat, engine, TestTables.TestTableType.HIVE_CATALOG, false,
-                formatVersion});
-            // test for vectorization=ON in case of ORC and PARQUET format with Tez engine
-            if (fileFormat != FileFormat.METADATA && "tez".equals(engine) && HiveVersion.min(HiveVersion.HIVE_3)) {
-              testParams.add(new Object[]{fileFormat, engine, TestTables.TestTableType.HIVE_CATALOG, true,
-                  formatVersion});
-            }
-          }
-        });
-      }
+      IntStream.of(2, 1).forEach(formatVersion -> {
+        testParams.add(new Object[]{fileFormat, HIVE_CATALOG, false, formatVersion});
+        // test for vectorization=ON in case of ORC and PARQUET format with Tez engine
```

Comment:
nit: it can be maybe:
```
// test for vectorization=ON in case of ORC and PARQUET format
```
fortunately you removed this whole execution engine specific thing, which is cool, so we can remove it from comment too

Reply:
done